tuna/miopen/db/miopen_tables.py

```diff
@@ -307,37 +308,7 @@ class FusionConfigTags(BASE, ConfigTagMixin):
   config = Column(Integer, ForeignKey("fusion_config.id"), nullable=False)
 
 
-class JobEnum(enum.Enum):
-  """Represents job_enum column in config table"""
-  # pylint: disable=invalid-name ; names represent entries in job_enum column
-  # pylint: disable=duplicate-code
-  new = 1
-  started = 2
-  running = 3
-  completed = 4
-  timeout = 5
-  error_status = 6
-  no_update = 7
-  errored = 8
-  bad_param = 9
-  error = 10
-  transfer_error = 11
-  eval_start = 12
-  evaluating = 13
-  evaluated = 14
-  evaluate_error = 15
-  compile_start = 16
-  compiling = 17
-  compiled = 18
-  compile_error = 19
-  not_applicable = 20
-  aborted = 21
-  not_tunable = 22
-  compiled_pend = 23
-  evaluated_pend = 24
-
-
-class FinStep(enum.Enum):
+class FinStep(JobEnum):
```

Comment:
FinStep should inherit from enum.Enum
This is a separate enumeration from JobEnum